packages/flutter/test/material/slider_theme_test.dart

```diff
@@ -102,10 +102,10 @@ void main() {
     final SliderThemeData sliderTheme = theme.sliderTheme;
 
     await tester.pumpWidget(_buildApp(sliderTheme, value: 0.5, enabled: false));
-    final RenderBox sliderBox = tester.firstRenderObject<RenderBox>(find.byType(Slider));
+    final MaterialInkController material = Material.of(tester.element(find.byType(Slider)));
```

Comment:
What is the reason for all these tests to change?

Reply:
The FocusableActionDetector that wraps the Slider inserts new semantics related render objects so it is more difficult to find the _RenderSlider, but the MaterialInkController can be used to test the same paint methods (This is how Switch does it)